PolicyProject1/src/main/java/com/example/demo/model/PolicyState.java

```diff
@@ -0,0 +1,5 @@
+package com.example.demo.model;
+
+public enum PolicyState {
```

Comment:
here again we are creating enum PolicyState which contains some constants.

Reply:
Every policy will go through these states. Thus, this constitutes lifecycle of Policy.

CREATED -> ACTIVE -> LAPSED -> RENEWED
(OR)
CREATED -> ACTIVE -> LAPSED -> CLOSED